api/refdataUserUrlUtil.ts

```diff
@@ -1,3 +1,3 @@
 export function getRefdataUserUrl(rdProfessionalApiPath: string, pageNumber?: string): string {
-  return `${rdProfessionalApiPath}/refdata/external/v1/organisations/users?size=50&page=${pageNumber}`;
+  return `${rdProfessionalApiPath}/refdata/external/v1/organisations/users?size=50&page=${pageNumber}&returnRoles=false&`;
```

Comment:
This is an expensive call with return roles not being set to false. On the users.component.ts there are 2 api calls to the same backend operation. With roles is a subset of the data and slow, without roles is the full list of users and a fraction of the time. For now I've updated this to not return roles since it's not needed to for the user list component.

I think there's some tech-debt to be looked at regarding getting the total count of users without having to make a second call